tutorials/plot_dipole_orientations.py

```diff
@@ -0,0 +1,196 @@
+"""
```

Comment:
Declare encoding ``# -*- coding: utf-8 -*-``. Is there a link for this tutorial in the web page?